### ISSUE TITLE
Exception in thread "main" java.lang.NullPointerException

### ISSUE THREAD

**Issue by palemieux** [skynav/ttt#181]
`java -jar c:\\bin\\ttxv\\ttt-ttxv-3.0-SNAPSHOT-jar-with-dependencies.jar --model imsc1 Animation014.ttml`

```
Exception in thread "main" java.lang.NullPointerException
        at com.skynav.ttv.verifier.imsc.IMSC1StyleVerifier.verifyTextOutlineAttributeItem(IMSC1StyleVerifier.java:252)
        at com.skynav.ttv.verifier.imsc.IMSC1StyleVerifier.verifyAttributeItem(IMSC1StyleVerifier.java:123)
        at com.skynav.ttv.verifier.ttml.TTML1StyleVerifier.verifyAttributeItems(TTML1StyleVerifier.java:518)
        at com.skynav.ttv.verifier.ttml.TTML1StyleVerifier.verify(TTML1StyleVerifier.java:499)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyStyleAttributes(TTML1SemanticsVerifier.java:982)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyParagraph(TTML1SemanticsVerifier.java:574)
        at com.skynav.ttv.verifier.imsc.IMSC1SemanticsVerifier.verifyParagraph(IMSC1SemanticsVerifier.java:632)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyBlock(TTML1SemanticsVerifier.java:917)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyDivision(TTML1SemanticsVerifier.java:549)
        at com.skynav.ttv.verifier.imsc.IMSC1SemanticsVerifier.verifyDivision(IMSC1SemanticsVerifier.java:600)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyBody(TTML1SemanticsVerifier.java:509)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyTimedText(TTML1SemanticsVerifier.java:341)
        at com.skynav.ttv.verifier.imsc.IMSC1SemanticsVerifier.verifyTimedText(IMSC1SemanticsVerifier.java:566)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verifyRoot(TTML1SemanticsVerifier.java:123)
        at com.skynav.ttv.verifier.ttml.TTML1SemanticsVerifier.verify(TTML1SemanticsVerifier.java:118)
        at com.skynav.ttv.verifier.imsc.IMSC1SemanticsVerifier.verify(IMSC1SemanticsVerifier.java:102)
        at com.skynav.ttv.app.TimedTextVerifier.verifySemantics(TimedTextVerifier.java:2088)
        at com.skynav.ttv.app.TimedTextVerifier.verify(TimedTextVerifier.java:2122)
        at com.skynav.ttv.app.TimedTextVerifier.verify(TimedTextVerifier.java:2250)
        at com.skynav.ttv.app.TimedTextVerifier.run(TimedTextVerifier.java:2327)
        at com.skynav.ttv.app.TimedTextVerifier.run(TimedTextVerifier.java:2293)
        at com.skynav.ttx.app.TimedTextTransformer.run(TimedTextTransformer.java:534)
        at com.skynav.ttx.app.TimedTextTransformer.run(TimedTextTransformer.java:530)
        at com.skynav.ttxv.app.TimedTextTransformingVerifier.main(TimedTextTransformingVerifier.java:77)
```
[Animation014.zip](https://github.com/skynav/ttt/files/728297/Animation014.zip)

**Comment by skynavga** [comment]
Fixed in 9369e6a9f56b193eb47b790976e1fd2eb7580861.

**Comment by skynavga** [comment]
Duplicate of #184.
